runtime/doc/diagnostic.txt

```diff
@@ -616,23 +623,20 @@ count({bufnr}, {opts})                                *vim.diagnostic.count()*
         (`table`) Table with actually present severity values as keys (see
         |diagnostic-severity|) and integer counts as values.
 
-disable({bufnr}, {namespace})                       *vim.diagnostic.disable()*
-    Disable diagnostics in the given buffer.
-
-    Parameters: ~
-      • {bufnr}      (`integer?`) Buffer number, or 0 for current buffer. When
-                     omitted, disable diagnostics in all buffers.
-      • {namespace}  (`integer?`) Only disable diagnostics for the given
-                     namespace.
+enable({bufnr}, {enable}, {opts})                    *vim.diagnostic.enable()*
+    Enables or disables diagnostics.
 
-enable({bufnr}, {namespace})                         *vim.diagnostic.enable()*
```

Comment:
`enable({bufnr}, {enable})` is the new signature. But the old `enable({bufnr}, {namespace})` is supported for back-compat, until deprecation cycle is finished.